src/main/resources/edu/cornell/kfs/module/ld/businessobject/datadictionary/PositionData.xml

```diff
@@ -57,45 +57,36 @@
+    <property name="beanName" value="PositionData-inquiryDefinition" />
+    <property name="fieldOverrides">
+      <list>
+          <!-- Bean override for new standalone page field presentation -->
+          <bean parent="FieldOverrideForListElementInsert"
+            p:propertyName="sections[0].fields"
+            p:propertyNameForElementCompare="name"
+            p:element-ref="PositionData-departmentId"
+            p:insertAfter-ref="PositionData-extension-orgCode"/>
 
-  <bean parent="DataDictionaryBeanOverride">
-      <property name="beanName" value="PositionData-inquiryDefinition" />
-      <property name="fieldOverrides">
-          <list>
-              <bean parent="FieldOverrideForListElementInsert">
-                  <property name="propertyName" value="inquirySections[0].inquiryFields" />
-                  <property name="propertyNameForElementCompare" value="attributeName" />
-                  <property name="element">
-                      <bean parent="FieldDefinition" p:attributeName="departmentId" />
-                  </property>
-                  <property name="insertAfter">
-                      <list>
-                          <bean parent="FieldDefinition" p:attributeName="extension.orgCode" />
-                      </list>
-                  </property>
-              </bean>
-          </list>
-      </property>
-  </bean>
-  
-  <bean parent="DataDictionaryBeanOverride">
-      <property name="beanName" value="PositionData-inquiryDefinition" />
-      <property name="fieldOverrides">
-          <list>
-              <bean parent="FieldOverrideForListElementInsert">
-                  <property name="propertyName" value="inquirySections[0].inquiryFields" />
-                  <property name="propertyNameForElementCompare" value="attributeName" />
-                  <property name="element">
-                      <bean parent="FieldDefinition" p:attributeName="extension.inactivationDate" />
-                  </property>
-                  <property name="insertAfter">
-                      <list>
-                          <bean parent="FieldDefinition" p:attributeName="extension.inactivationDate" />
-                      </list>
-                  </property>
-              </bean>
-          </list>
-      </property>
+          <bean parent="FieldOverrideForListElementInsert"
+            p:propertyName="sections[0].fields"
+            p:propertyNameForElementCompare="name"
+            p:insertAfter-ref="PositionData-extension-inactivationDate"/>
+
+          <!-- Bean override for legacy modal style presentation -->
+          <bean parent="FieldOverrideForListElementInsert"
+            p:propertyName="inquirySections[0].inquiryFields"
+            p:propertyNameForElementCompare="attributeName"
+            p:element-ref="PositionData-departmentId"
+            p:insertAfter-ref="PositionData-extension-orgCode"/>
```

Comment:
When overriding legacy inquiry setup, the "element" and "insertAfter" properties have to contain "FieldDefinition" child beans; they can't contain the regular attribute definitions. Please adjust the legacy inquiry setup accordingly.